web/src/beta/components/Modal/index.tsx

```diff
@@ -1,54 +1,163 @@
-import { ReactNode } from "react";
+import { ReactNode, useState } from "react";
 
 import Wrapper from "@reearth/beta/components/Modal/ModalFrame";
-import Text from "@reearth/beta/components/Text";
-import { styled, useTheme } from "@reearth/services/theme";
+import { styled } from "@reearth/services/theme";
+
+type Size = "sm" | "md" | "lg";
+
+export type SidebarTab = {
+  id: string;
+  label: string;
+  content: ReactNode;
+  tabButton1?: ReactNode;
+  tabButton2?: ReactNode;
+};
 
 type Props = {
   className?: string;
-  title?: string;
-  size?: "sm" | "md" | "lg";
+  modalTitle?: string;
+  size?: Size;
   button1?: ReactNode;
   button2?: ReactNode;
   children?: ReactNode;
   isVisible?: boolean;
   onClose?: () => void;
+  showSidebar?: boolean;
```

Comment:
Like I said yesterday, this isn't necessary. If we are passing sidebarTabs, we know it needs a sidebar.